en/components/grid-row-selection.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Grid Row Selection - Grid Feature
+_description: The Grid Row Selection provides a mechanism for selecting a full record shown in the Grid.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid Row Selection
+
+Use the Grid Row Selection to let one select complete rows at once via a special selection column with checkboxes that appears as the leftmost column of the Grid. The Grid Row Selection is visually identical to the Row Selection mode of the [Ignite UI for Angular Grid Selection Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/selection.html#grid-row-selection)
```

Comment:
Current: Use the Grid Row Selection to let one select complete rows 
Suggestion: remove -let one- which sounds a little bit strange in the complete sentence => Use the Grid Row Selection to select complete rows